### ISSUE TITLE
Bottom nav bar is blocking out Action Bar on `member/edit`(smaller screens)

### ISSUE THREAD

**Issue by drillprop** [Joystream/atlas#2082]
## Scope

- [ ] Hide bottom nav on small screens where action bar is shown

The Action bar component ([Figma docs](https://www.figma.com/file/Pf31tuYpozYmpq163U2ho8/Web-components?node-id=7688%3A149633)) should _always_ be displayed above the Bottom navigation component ([Figma docs](https://www.figma.com/file/Pf31tuYpozYmpq163U2ho8/Web-components?node-id=3173%3A136439)). In other words, the Bottom navigation should be hidden (although, technically _hiding_ is not a necessity, it can be just displayed with a lower z-index, IMHO) whenever the Action bar is around.

⚠️The Bottom navigation component is visible only up until the MD breakpoint (1024px).

## Context

**Describe the bug**

As the title says.

**To Reproduce**

1. Go to '/member/edit'

**Expected behavior**

The design specifies that the bottom nav should be hidden in this case. 
https://www.figma.com/file/yhZpTHdf1sxJx13uRZ71GV/Membership-profile?node-id=968%3A154096

**Screenshots**
![image](https://user-images.githubusercontent.com/51168865/153000854-6d3b45dc-1360-4380-ac75-6a42c5d64744.png)

**Device (please complete the following information):**

- OS: macOS Monterey]
- Browser: Chrome

**Comment by dmtrjsg** [comment]
Hey @toiletgranny what should be done in this case? The designs do not include the bottom nav..

**Comment by toiletgranny** [comment]
Hey @dmtrjsg, the design specifies that the bottom nav should be hidden in this case.

**Comment by dmtrjsg** [comment]
@toiletgranny is "this case" denotes the case when action bar is visible? Or always for the smallest screens? Pls clarify :)

**Comment by toiletgranny** [comment]
@dmtrjsg 

> @toiletgranny is "this case" denotes the case when action bar is visible?

Yes, the Action bar component ([Figma docs](https://www.figma.com/file/Pf31tuYpozYmpq163U2ho8/Web-components?node-id=7688%3A149633)) should _always_ be displayed above the Bottom navigation component ([Figma docs](https://www.figma.com/file/Pf31tuYpozYmpq163U2ho8/Web-components?node-id=3173%3A136439)). In other words, the Bottom navigation should be hidden (although, technically _hiding_ is not a necessity, it can be just displayed with a lower z-index, IMHO) whenever the Action bar is around.

> Or always for the smallest screens?

Because of the above, I would say that the answer to the above is also _yes_. Let's keep in mind that the Bottom navigation component is visible only up until the MD breakpoint (1024px).
